app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor.keys():
+            raise NotVaccinatedError("Visitor should be vaccinated")
+        elif (
+                datetime.date.today()
+                > visitor.get("vaccine").get("expiration_date")

```

Comment:
But here .get is not needed. We know that vaccine exist. And if vaccine exits expiration_date also exists